### DISCUSSION TITLE
Unblocking a domain from DNS Filters

### DISCUSSION THREAD

**Discussion by xiki808** [AdguardTeam/AdGuardHome#3268] [Q&A] [answered]
I am writing here to get a direction and if possible help from the community with a domain my company is building a product around, but which we found it is blocked from a number of DNS filters.

I have noticed this with my current AdGuard Home setup and when "Use AdGuard parental control web service" is on.

The domain name is time2play.com and we found out that this same domain was used for adult content before.

How can we clean this domain name and stop it from being blocked by a number of DNS filters and/or ad blockers. Can you do something about this? Do you have some guides or directions for me on what I should do to achieve this?

Thank you!

**Comment by ZeroClover** [comment]
You need to submit an issue to lists that have blocked this domain.

However, if a significant number of lists block the domain, it may be a difficult task to completely remove your domain from all lists, considering that not everyone updates their local device lists promptly and that some commercial products rely on these public lists. Using another TLD (such as .io) may be a better option.

**Comment by Disqu3-mirOir-qt** [comment, marked as answer]
hi y can do it filters rule @@||domain.com$important

**Comment by xiki808** [comment]
@ZeroClover Thank you for your insights, this is what we are doing right now

Would you be able to point me out which list is used when having the parental control service on?
